src/components/Missions/Mission.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Table, Badge, Button } from 'react-bootstrap';
+import { fetchMissions } from '../../redux/Missions/missionReducer';
+import TableHeader from './TableHeader';
+import '../../css/Table.css';
+
+const Mission = () => {
+  const items = useSelector((state) => state.missionsReducer);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchMissions());
+  }, []);
```

Comment:
- [ ] I think it's better to move this to the App component and pass the items to the mission component as props to prevent fetching the data at every re-render of the Mission component. As it is required to only fetch the data once.